stock_picking_mto/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,2 @@
+* David Vidal <david.vidal@tecnativa.com>
+* Pedro Baeza <pedro.baeza@tecnativa.com>
```

Comment:
`Pedro M. Baeza`, without emails, and with the company name and link, please :angel: